### ISSUE TITLE
Support Unix sockets in --url

### ISSUE THREAD

**Issue by jo-so** [sequelize/cli#695]
I would like to access my database via Unix socket. This requires to use a path as hostname, but not all path characters are valid for a hostname. Rust's r2d2 database driver uses an [URL-encoded form of the hostname](https://github.com/sfackler/rust-postgres#connecting) like: `postgres://joerg@%2Frun%2Fpostgresql:5433/codimd?application_name=CodiMD`. Maybe you could support this, too. Or use a query parameter for this.

**Comment by bogdan** [comment]
Sequelize itself supports a socket given as a host parameter like:

```
postgres://postgres@/unstoppable_website_development?host=/tmp
```

But not sequelize-cli

Its weird that sequelize-cli doesn't reuse that code from sequelize itself.

**Comment by jo-so** [comment]
I think it's up to the devs to implement this request. This issue stalled, because the devs didn't react.

**Comment by sushantdhiman** [comment]
Tagging this as feature request

> I think it's up to the devs to implement this request. This issue stalled, because the devs didn't react.

@jo-so I don't have infinite time at my hand. I will appreciate a PR with proper test

**Comment by jo-so** [comment]
Does the feature tag keep away the stale bot?

**Comment by sushantdhiman** [comment]
Yes, but issues will be disabled soon and only PRs will be accepted, so it wont matter if issue is closed or open
